tests/test_ecs/test_ecs_boto3.py

```diff
@@ -47,6 +47,40 @@ def test_create_cluster_with_setting():
     assert cluster["serviceConnectDefaults"] == {"namespace": "ns"}
 
 
+@mock_aws
+def test_create_cluster_with_running_tasks():
+    running_tasks_count = 3
```

Comment:
I haven't run the tests yet - but I know both will fail in ServerMode, because that runs Moto in a different process and the environment variables will not be applied.

We usually just skip these tests if that's the case:
```python
if not settings.TEST_DECORATOR_MODE:
    raise SkipTest("Can't set environment variables in server mode")
```